Automattic/jetpack
# Notifications - ensure cache buster is available for notifications script.

Fixes # pe7F0s-1LH-p2

## Proposed changes:

* Ensures we add the jetpack notes cache buster to the wpNotesArgs window object that the wpcom-notes-admin-bar script uses to build the version number for notifications.
* Before this change, all jetpack and atomic sites were requesting notifications html file from widgets with param `v=none`. This can cause inconsistency in the version of the html file and the scripts it references, causing outdated functionality, missing bug fixes, etc.
* After this change, jetpack sites should be requesting this file with a version number corresponding to the JETPACK_NOTES__CACHE_BUSTER which is updated at each jetpack version. This should ensure sites will not receive long outdated notifications files/scripts.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Run this branch of jetpack with notes module enabled. (activate this branch with the jetpack beta tester plugin on a woa site)
* Load the site (front end seems fine)
* Inspect the network tab and search for 'notifications'.
* verify the request to widgets notifications is no longer supplying the `v=none` param and now has a value associated with the jetpack notes cache buster.
<img width="948" alt="Screenshot 2024-05-03 at 12 11 29 PM" src="https://github.com/Automattic/jetpack/assets/28742426/64cf0ab5-62e9-4d4c-aa2e-ce6c865ba282">
* Verify notifications continue to work as expected.
